data/README.md

```
@@ -84,3 +84,14 @@ For example, for PreProd, that means removing all immutable chunks after `03149`
 3. Start your node.
 
 Both Ogmios and the fetch script will automatically connect to the node once ready and capture data from it when it becomes available. This should terminate once all the points have been processed.
+
+4. Dump chain headers as CBOR
+
+Go to home directory `cd ..` and make fetch-chain-header :
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)